BDArmory/UI/BDATargetManager.cs

```diff
@@ -550,7 +556,7 @@ public static TargetSignatureData GetHeatTarget(Vessel sourceVessel, Vessel miss
             if (priorHeatScore > 0) // Flares can only decoy if we already had a target
             {
                 flareData = GetFlareTarget(ray, scanRadius, highpassThreshold, lockedSensorFOVBias, lockedSensorVelocityBias, priorHeatTarget);
-                flareSuccess = ((!flareData.Equals(TargetSignatureData.noTarget)) && (flareData.signalStrength > highpassThreshold));
+                flareSuccess = ((!flareData.Equals(TargetSignatureData.noTarget)) && (flareData.signalStrength * flareEffectivity > highpassThreshold));
```

Comment:
flareEffectivity here won't have the desired effect. This only determines if the flare is actually being spotted by the seeker head.